reference/5.1/Microsoft.PowerShell.Core/About/about_numeric_literals.md

```diff
@@ -0,0 +1,190 @@
+---
+ms.date:  04/09/2018
+schema:  2.0.0
+locale:  en-us
+keywords:  powershell,cmdlet
+title:  About numeric literals
+---
+# About numeric literals
+
+There are two kinds of numeric literals: integer and real. Both can have type
+and multiplier suffixes.
+
+## Integer literals
+
+Integer literals can be written in decimal or hexadecimal notation. Hexadecimal
+literals are prefixed with "0x" to distinguish them from decimal numbers.
```

Comment:
Should "0x" be in a codefence as a prefix since all the suffix examples are?

Reply:
Fixed